src/tests/multihost/ipa/test_misc.py

```diff
@@ -79,3 +81,34 @@ def test_sssdConfig_remove_Domains(self, multihost):
                                                    raiseonerr=False)
             assert cmd1.returncode == 0
             assert cmd2.returncode == 0
+
+    def test_filter_groups(self, multihost, add_group_member, backupsssdconf):
+        """
+        :title:  filter_groups option partially filters the group from id
+        output of the user because gidNumber still appears in id output
+        :id: 8babb6ee-7141-4723-a79d-c5cf7879a9b4
+        :description:
+         filter_groups option partially filters the group from 'id' output
+         of the user because gidNumber still appears in 'id' output
+        :bugzilla:
```

Comment:
I think that this is a customer scenario test as it has a CC attached to the bug.